package.json

```diff
@@ -138,63 +135,63 @@
     "@types/react-dom": "^17.0.14",
     "@types/react-modal": "^3.13.1",
     "alasql": "1.7.3",
-    "benchmark": "2.1.4",
+    "benchmark": "^2.1.4",
     "braces": "^3.0.2",
     "cross-env": "^7.0.3",
     "diff": "^5.0.0",
     "downshift": "^6.1.7",
-    "eslint": "^7.18.0",
+    "eslint": "^8.8.0",
     "eslint-gitignore": "^0.1.0",
-    "eslint-plugin-jest": "26.1.5",
-    "eslint-plugin-node": "11.1.0",
+    "eslint-plugin-jest": "^26.1.5",
+    "eslint-plugin-node": "^11.1.0",
     "extend": "^3.0.2",
-    "foreman": "3.0.1",
-    "fzf": "0.5.1",
-    "history": "5.2.0",
-    "html-validate": "6.8.0",
+    "foreman": "^3.0.1",
+    "fzf": "^0.5.1",
+    "history": "^5.3.0",
+    "html-validate": "^6.8.0",
     "husky": "^7.0.4",
     "ignore-loader": "^0.1.2",
     "jest-junit-reporter": "^1.1.0",
     "jest-puppeteer": "6.1.0",
     "jsdom": "^19.0.0",
     "lint-staged": "^12.4.1",
-    "node-dev": "7.4.3",
+    "nodemon": "^2.0.15",
     "pegjs": "^0.10.0",
     "prettier": "^2.6.2",
     "prettier-plugin-packagejson": "^2.2.17",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
+    "pretty-quick": "^3.1.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
     "react-is": "^18.1.0",
-    "react-refresh": "0.13",
-    "react-router": "6.3.0",
-    "react-router-dom": "6.3.0",
-    "react-scripts": "4.0.3",
+    "react-refresh": "^0.13.0",
+    "react-router": "^6.3.0",
+    "react-router-dom": "^6.3.0",
+    "react-scripts": "^5.0.0",
     "react-simple-code-editor": "0.11.0",
     "rehype-minify-whitespace": "^4.0.5",
     "remark-prettier": "^1.3.0",
-    "rough-notation": "0.5.1",
+    "rough-notation": "^0.5.1",
     "sass": "^1.51.0",
     "source-map-explorer": "^2.5.2",
     "style-dictionary": "^3.7.0",
-    "stylelint": "^13.13.1",
+    "stylelint": "^14.5.0",
     "stylelint-a11y": "^1.2.3",
     "stylelint-config-prettier": "^9.0.3",
-    "stylelint-config-recommended": "^5.0.0",
-    "stylelint-config-sass-guidelines": "^8.0.0",
-    "stylelint-prettier": "^1.2.0",
-    "stylelint-scss": "^3.21.0",
-    "swr": "^0.5.7",
-    "ts-loader": "^8.1.0",
-    "ts-node": "^10.7.0",
+    "stylelint-config-recommended": "^7.0.0",
+    "stylelint-config-sass-guidelines": "^9.0.1",
+    "stylelint-prettier": "^2.0.0",
+    "stylelint-scss": "^4.1.0",
+    "swr": "^1.2.2",
+    "ts-loader": "^9.2.6",
     "typescript": "^4.6.4",
     "use-debounce": "^8.0.0",
-    "webpack": "4.44.2",
+    "webpack": "^5.68.0",
     "webpack-cli": "^4.9.2",
     "webpack-dev-middleware": "^5.3.1",
     "webpack-hot-middleware": "^2.25.1",
     "webpack-node-externals": "^3.0.0"
   },
   "engines": {
-    "node": ">=12.11.0 <17.0.0"
+    "node": "^14.13.1 || >=16.0.0"
```

Comment:
@schalkneethling Do you still remember why you bumped Node.js to 14+, dropping support for Node.js 12?

Reply:
I believe with ESM we need at least 14, maybe even 16 to be honest.